com.unity.render-pipelines.core/Editor/Volume/VolumeEditor.cs

```diff
@@ -9,6 +10,33 @@ namespace UnityEditor.Rendering
     [CustomEditor(typeof(Volume))]
     sealed class VolumeEditor : Editor
     {
+        static class Styles
+        {
+            public static readonly GUIContent mode = EditorGUIUtility.TrTextContent("Mode", "A global volume is applied to the whole scene.");
+            public static readonly GUIContent[] modes =
+                {
+                    EditorGUIUtility.TrTextContent("Global"),
+                    EditorGUIUtility.TrTextContent("Local")
+                };
+
+            public static readonly GUIContent addBoxCollider = EditorGUIUtility.TrTextContent("Add a Box Collider");
+            public static readonly GUIContent sphereBoxCollider = EditorGUIUtility.TrTextContent("Add a Sphere Collider");
+            public static readonly GUIContent capsuleBoxCollider = EditorGUIUtility.TrTextContent("Add a Capsule Collider");
+            public static readonly GUIContent meshBoxCollider = EditorGUIUtility.TrTextContent("Add a Mesh Collider");
+
+            public static readonly GUIContent addColliderFixMessage = EditorGUIUtility.TrTextContentWithIcon("Add a Collider to this GameObject to set boundaries for the local Volume.", CoreEditorStyles.iconWarn);
+            public static readonly GUIContent disableColliderFixMessage = EditorGUIUtility.TrTextContentWithIcon("Global Volumes do not need a collider. Disable or remove the collider.", CoreEditorStyles.iconWarn);
+            public static readonly GUIContent enableColliderFixMessage = EditorGUIUtility.TrTextContentWithIcon("Local Volumes need a collider enabled. Enable the collider.", CoreEditorStyles.iconWarn);
+
+            public static readonly GUIContent profileInstance = EditorGUIUtility.TrTextContent("Profile (Instance)", "A copy of a profile asset.");
+            public static readonly GUIContent profile = EditorGUIUtility.TrTextContent("Profile", "A reference to a profile asset.");
```

Comment:
Today I created a PR (#5000) updating those tooltips. These need to be updated to use the same.